veterinary-pathology.csl

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>Veterinary Pathology</title>
+    <id>http://www.zotero.org/styles/veterinary-pathology</id>
+    <link href="http://www.zotero.org/styles/veterinary-pathology" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver-superscript" rel="template"/>
+    <link href="https://mc.manuscriptcentral.com/societyimages/vetpath/Veterinary%20Pathology-Instructions%20to%20Authors%202016-4-18.pdf" rel="documentation"/>
```

Comment:
(@damnation333, in general, I try avoiding linking to documentation URLs that have a date in them, such as "... 2016-4-18.pdf" here, since they're likely to become dead links relatively quick. https://us.sagepub.com/en-us/nam/veterinary-pathology/journal201966#submission-guidelines might be more stable in this case. [I'll make the change myself])